spec/src/main/asciidoc/jakarta-data-introduction.asciidoc

```diff
@@ -0,0 +1,29 @@
+= Introduction
+
+Jakarta Data provides an API that makes data access easy. A
+Java developer can split the persistence from the model with several features,
+such as the ability to compose custom query methods on a Repository interface where the framework
+will implement it.
+
+== Introduction
+
+There is no doubt about the importance of data around the application. We often talk about a stateless application, the data where we delegate the application's state. If the architecture is the mind, the information is the heart.
+
+Dealing with a database is one of the biggest challenges within a software architecture. In addition to choosing one of several options on the market, it is necessary to consider the persistence integrations. Jakarta Data claims to make life easier for developers with Java.
+
```

Comment:
"Jakarta Data claims to make life easier for developers with Java."
could be reworded as:
"Jakarta Data makes life easier for Java developers."